src/listener/alerts-listener.js

```diff
@@ -54,7 +54,7 @@ export default class AlertsListener extends Listener {
     let time = msg.time;
 
     this.client.Alerts.getFilteredAlerts(status, state, time).then((res) => {
-      if (res.count === 0) {
+      if (res.members.length === 0) {
```

Comment:
Why change this here?  Using the res.count is probably fine.

Reply:
Basically because the response does not have the count attribute, it has only a member array.